source/production/F0.Minesweeper.Components/Game.razor.css

```diff
@@ -0,0 +1,2 @@
+#f0-game {
+}
```

Comment:
Maybe more of a question to discuss: Do we want to keep those empty CSS files/classes/selectors/etc. or should we just add them when necessary?

Reply:
To be honest I just forgot to remove this again. I will for now remove it since I personally think they shouldn't be there when there's no need for them. When you create a new component there's also no `css` file automatically created so I'd stick to the standard. 
Let's discuss this in the next session!